packages/redux-slice-factory/src/slice-factories/create-model-slice/CreateModelSlice.ts

```diff
@@ -149,6 +149,13 @@ function createModelSlice<
         selectError: createSelector(selectSliceState, (sliceState) => sliceState.error),
         selectLastModified: createSelector(selectSliceState, (sliceState) => sliceState.lastModified),
         selectLastHydrated: createSelector(selectSliceState, (sliceState) => sliceState.lastHydrated),
+        selectShouldFetch: (reFetchOnError = false, overwriteAllData = false) => createSelector(
```

Comment:
i think it might make more sense to make this configurable in the constructor, like we let someone basically create their own version of the selector function and we just pass that to your create selector